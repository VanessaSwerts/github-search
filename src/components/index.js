import CustomHeader from './Header/Header'
import CustomFooter from './Footer/Footer'
import Error from './Error/Error'
import Loading from './Loading/Loading'

export {
  CustomHeader,
  CustomFooter,
  Error,
  Loading
}