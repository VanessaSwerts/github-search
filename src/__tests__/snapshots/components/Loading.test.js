import React from 'react'
import { render } from '@testing-library/react'
import { Loading } from '../../../components'

describe('Testes de Snapshot da Loading', () => {

  test('Renderização da tela Loading', () => {
    const { asFragment } = render(<Loading />)

    expect(asFragment(<Loading />)).toMatchSnapshot()
  })

}) 