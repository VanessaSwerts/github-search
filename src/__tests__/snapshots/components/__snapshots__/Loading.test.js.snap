// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes de Snapshot da Loading Renderização da tela Loading 1`] = `
<DocumentFragment>
  <div
    class="card-loading"
  >
    <i
      class="fas fa-spinner"
    />
    <span
      data-testid="message-loading"
    >
      Carregando ...
    </span>
  </div>
</DocumentFragment>
`;
